order/manager.go

```diff
@@ -142,30 +145,72 @@ func (m *Manager) PrepareOrder(ctx context.Context, order Order,
 		return nil, err
 	}
 
-	// Grab the additional information needed from our local node.
-	nextMultiSigKey, err := m.cfg.Wallet.DeriveNextKey(
-		ctx, int32(keychain.KeyFamilyMultiSig),
-	)
-	if err != nil {
-		return nil, fmt.Errorf("unable to derive "+
-			"multi sig key: %v", err)
-	}
-	order.Details().MultiSigKeyLocator = nextMultiSigKey.KeyLocator
-	var multiSigKey [33]byte
-	copy(multiSigKey[:], nextMultiSigKey.PubKey.SerializeCompressed())
-	info, err := m.cfg.Lightning.GetInfo(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("unable to get local node info: %v", err)
-	}
-	nodeAddrs, err := parseNodeUris(info.Uris)
-	if err != nil {
-		return nil, fmt.Errorf("unable to parse node uris: %v", err)
+	params := &ServerOrderParams{}
+	bid, isBid := order.(*Bid)
+	isSidecar := isBid && bid.SidecarTicket != nil
+
+	// Using a sidecar ticket with a bid order means we won't be receiving
+	// the channel ourselves and are instead leasing a channel for another
+	// node. Therefore we need to add the other node's identity and multisig
+	// public key in the bid order we send to the auctioneer.
+	if isSidecar {
```

Comment:
See comment re the use case of people wanting balanced chanenls from the getgo, w/o no third party involved. We've seen many people on reddit and those telegram groups are doing a sort of "trust fall" exercise, which we can make more trust minimized by allowing them to use our existing execution engine for this purpose.